bin/staging_multideploy.sh

```diff
@@ -17,5 +17,6 @@ helm upgrade $CLEANED_BRANCH_NAME \
   --set image.repository=$DOCKER_REPOSITORY \
   --set image.tag=$IMAGE_TAG \
   --set dashboard.enabled=false \
+  --set-string pingdomIPs=$PINGDOM_IPS \
```

Comment:
Is this dynamic staging. If so do we need pingdom access? Is it simpler to keep?

Reply:
yes this dynamic staging and no we don't need pingdom access on this environment. However it makes writing the config writing simpler